spec/unit/generator/xcconfig/xcconfig_helper_spec.rb

```diff
@@ -241,6 +241,93 @@ module XCConfig
 
         #---------------------------------------------------------------------#
 
+        describe 'concerning settings for file accessors' do
```

Comment:
lol @paulb777 I think we wrote almost the same tests (https://github.com/CocoaPods/CocoaPods/pull/6988)...I think the change we both did is different however :)

Reply:
This change basically ensures test spec paths dont end up in aggregates

Reply:
:) Yeah, if I understand correctly, this change is preventing test spec paths from ending up in aggregates, while #6988 was about making sure that paths were properly propagated aggregates including static frameworks.

Reply:
Yeap exactly.